projectile.el

```diff
@@ -1095,6 +1095,7 @@ Files are returned as relative paths to the project ROOT."
 (defun projectile-get-sub-projects-command ()
   (let ((vcs (projectile-project-vcs)))
     (cond
+     ((not projectile-git-submodule-command) "")
      ((eq vcs 'git) projectile-git-submodule-command)
```

Comment:
I think you actually need to add an if on the next line, give that all of this is `git`-specific. Or maybe better - make the receivers of this method handle `nil` instead of an empty string. I guess this change from nil to "" broke the functionality in the first place.

Reply:
> make the receivers of this method handle nil instead of an empty string.

If you're willing to take that, it is definitely better!

Reply:
Fixed up. Much simpler than I expected. And added extra tests cause why not.